react/components/EXPERIMENTAL_TableTree/Tree/Node.tsx

```diff
@@ -35,7 +41,7 @@ const Node: FC<NodeProps> = ({
     : undefined
 
   const clickableCell = hasChildren
-    ? !!clickableRow
+    ? clickableRow
```

Comment:
All ok without the double exclamation mark? Sounds like `clickableRow` is not a bool.

Reply:
yeah, because it is being used in the ternary expression, so it will be coarsed to a bool anyway